src/decaylanguage/data/__init__.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
-try:
-    from importlib.resources import open_text
-except ImportError:
+import sys
+
+if sys.version_info < (3, 7):
```

Comment:
It's better to compare with sys.version_info. It:
* Tells a reader exactly where one works vs. the other
* Works with static tools, like mypy
* Can support using a backport after it was introduced (if we use the newer 3.8/3.9  API of importlib.resouces, for example).

Reply:
Makes sense :-).